docs/cimpl6LanguageReference.md

```diff
@@ -195,21 +177,21 @@ Primitives are data types, distinguished by starting with a lower case letter. C
 * [`positiveInt`](https://www.hl7.org/fhir/datatypes.html#positiveInt)
 * [`xhtml`](https://www.hl7.org/fhir/narrative.html#xhtml)
 
->**Note:** CIMPL does not have an explicit _Reference_ data type. References are determined during the [mapping process](#mapping-to-references).
+>**Note:** Unlike FHIR, CIMPL does not have an explicit [**Reference**](https://www.hl7.org/fhir/references.html) data type. References are determined during the [mapping process](#mapping-to-references).
 
 ### Concept Codes
 
-CIMPL uses a single primitive type, `concept` to represent coded terms from controlled vocabularies. A concept is comprised of code system, code, and optional display text. The grammar for specifying concepts is:
+CIMPL uses a single primitive type, `concept` to represent coded terms from controlled vocabularies. A concept is comprised of code system, code, and optional display text. The grammar for specifying concepts follows the pattern:
 
-> `SYSTEM#code "Display text"`
+_SYSTEM#code "Display text"_
```

Comment:
Are SYSTEM, code, and Display text substitutions?  If so, then either bolded italics as in lines 82-84 or not?  Also - the pound sign  s/b in code block.

Reply:
It's a code with substitutions, so it should be: <code><i>SYSTEM</i>#<i>code</i> "<i>Display text</i>"</code>.